test_scripts/API/VehicleData/common.lua

```diff
@@ -7,20 +7,17 @@ local runner = require('user_modules/script_runner')
 local utils = require("user_modules/utils")
 local json = require("modules/json")
 local SDL = require("SDL")
-local apiLoader = require("modules/api_loader")
 local color = require("user_modules/consts").color
+local ah = require("user_modules/api/APIHelper")
+local tdg = require("user_modules/api/APITestDataGenerator")
 
 --[[ General configuration parameters ]]
 runner.testSettings.isSelfIncluded = false
 config.defaultProtocolVersion = 2
+config.zeroOccurrenceTimeout = 1000
```

Comment:
What does zero occurrence mean? What is this used for?

Reply:
This parameter defines a timeout for SDL to wait for an events that should not occurs.
For example in ATF script we're expecting `OnVehicleData` should not be transfered by SDL to Mobile.
In this case we register expectation with `Times(0)`:
```
common.getMobileSession():ExpectNotification("OnVehicleData"):Times(0)
```
Having this ATF will wait until timeout defined by `zeroOccurrenceTimeout` parameter is expired and then finish test step.

Decreasing default value from `2000` to `1000` ms allows us to run VD test scripts faster.